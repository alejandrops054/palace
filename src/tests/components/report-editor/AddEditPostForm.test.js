/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import { shallow } from 'enzyme'
import AddEditForm from '../../../components/report-editor/AddEditPostForm'

describe('Prueba componente UI <AddEditPostForm/>', () => {
    test('Componente no recibe props', () => {

        const wrapper = shallow(
            <AddEditForm />
        );

        expect(wrapper).toMatchSnapshot();
    })
})
