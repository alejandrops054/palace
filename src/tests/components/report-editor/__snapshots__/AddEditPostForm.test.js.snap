// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba componente UI <AddEditPostForm/> Componente no recibe props 1`] = `
<Fragment>
  <AddEditForm />
</Fragment>
`;
